app/views/layouts/application.html.erb

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8"/>
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title><%= @page_title || _('openSUSE Software') %></title>
+    <title><%= "#{@page_title} | #{_('openSUSE Software')}" || _('openSUSE Explore') %></title>
```

Comment:
what is the expected result of this? :thinking:

Reply:
If no @page_title is defined, it shows default "Explore", otherwise it combines page title with "openSUSE Software", so it's always obvious what page user is on from Titlebar (as it wasn't obvious for for example "Search" title beforehand)

Reply:
@hellcp but I think that is not what it is doing. If `@page_title` is defined, let say it is `openSUSE Leap 15`, it will show:

`openSUSE Leap 15 | openSUSE Software`

and if it is not defined it will show:

`" | openSUSE Software"`

I think that what you want to write is: :wink: 

` @page_title ? "#{@page_title} | #{_('openSUSE Software')}" : _('openSUSE Explore')`

I think this happens in other parts as well. :thinking: